key_teleop/key_teleop/key_teleop.py

```diff
@@ -124,7 +125,13 @@ def __init__(self, interface):
         super().__init__('key_teleop')
 
         self._interface = interface
-        self._pub_cmd = self.create_publisher(Twist, 'key_vel')
+
+        self._twist_stamp_enabled = self.declare_parameter('twist_stamped_enabled', False).value
```

Comment:
```suggestion
        self._publish_stamped_twist = self.declare_parameter('publish_stamped_twist', False).value
```
I'd suggest renaming this variable and the parameter name for making it even clearer